bin/cloud-local.sh

```diff
@@ -38,8 +38,7 @@ function download_packages {
   if [[ ! -z ${pkg_pre_download+x} ]]; then
     # Does that folder actually exist?
     if [[ -d ${pkg_pre_download} ]] ; then
-      test -d ${CLOUD_HOME}/pkg || rmdir ${CLOUD_HOME}/pkg
-      test -h ${CLOUD_HOME}/pkg && rm ${CLOUD_HOME}/pkg
+      rm -f ${CLOUD_HOME}/pkg
```

Comment:
you'll likely need ``-rf`` or ``rmdir`` here? any reason not to keep the test as is? that way no errors are thrown if the directory does not exist I think...

Reply:
also the previous ``test -h`` unlinked and relinked the ``pkg_pre_download`` dir so that if it changes the script works?

Reply:
So here was the problem i hit...basically if you have previously tried to download packages then want to enable the shared dir it fails:
```
ahulbert@nuc16:/tmp/cloud-local$ bin/cloud-local.sh init
rm: cannot remove '/tmp/cloud-local/pkg': Is a directory
ln: failed to create symbolic link '/tmp/cloud-local/pkg/packages': File exists
```

Reply:
My feeling is primarily that the user should decide what they want to do with the contents of the existing folder... e.g. copy into synology? just plain delete? rename to folder to keep as other stuff?

Reply:
yeah that sounds like a good idea. Lets not do an RM at all then? we can instead just set a variable to be the pkg dir and pull stuff out of there? That would require making something new like setting a ``cl_pkg_dir`` that if the pre download is not set would become ``${CLOUD_HOME}/pkg`` ? that way we don't remove anything and just use the new dir as is and no symlinks. nothing to delete/remove